### PR TITLE
Remove undeclared test outputs after zipping on Windows

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

**Comment by fmeum** [comment]
Stacked on #16973

**Comment by fmeum** [comment]
@tjgq Could you review?

**Comment by fmeum** [comment]
No longer stacked.

File: tools/test/windows/tw.cc [hunk 1]
```diff
@@ -1309,7 +1322,8 @@ bool ArchiveUndeclaredOutputs(const UndeclaredOutputs& undecl) {
   return GetFileListRelativeTo(undecl.root, &files) &&
          (files.empty() ||
           (CreateZip(undecl.root, files, undecl.zip) &&
-           CreateUndeclaredOutputsManifest(files, undecl.manifest)));
+           CreateUndeclaredOutputsManifest(files, undecl.manifest) &&
+           RemoveRelativeRecursively(undecl.root, files)));
```
[the remark this answers is not in the document]
Comment (fmeum) [reply tools/test/windows/tw.cc]: Changed

**Review by tjgq — APPROVED** [review]

**Pull request closed without merging**